pkgs/secretlint/secretlint/registry.yaml

```diff
@@ -0,0 +1,99 @@
+packages:
+  - type: github_release
+    repo_owner: secretlint
+    repo_name: secretlint
+    description: Pluggable linting tool to prevent committing credential
+    version_constraint: "false"
+    version_overrides:
+      - version_constraint: semver("<= 0.3.0")
+        no_asset: true
+      - version_constraint: semver("<= 0.7.3")
+        asset: secretlint_{{.OS}}_{{.Arch}}
+        format: raw
+        rosetta2: true
+        windows_arm_emulation: true
+        replacements:
+          linux: Linux
+          windows: Windows
+        overrides:
+          - goos: windows
+            asset: secretlint_{{.OS}}_NT_{{.Arch}}
+            replacements:
+              amd64: x64
+        supported_envs:
+          - windows
+          - x64
```

Comment:
```suggestion
          - amd64
```

x64 isn't supported.
I guess amd64 is correct.

Reply:
But darwin should be supported too.

```yaml
        supported_envs:
          - darwin
          - windows
          - amd64
```

https://github.com/aquaproj/aqua-registry/pull/22817#issuecomment-2106892568

Reply:
> But darwin should be supported too.

🤔 Sorry, this was wrong.

https://github.com/secretlint/secretlint/releases/tag/v0.7.3

Reply:
Fixed. 7eb99e32607a423d72924019f96dee9654528656